sklearn/metrics/_ranking.py

```diff
@@ -855,11 +855,11 @@ def precision_recall_curve(y_true, probas_pred, *, pos_label=None, sample_weight
     >>> precision, recall, thresholds = precision_recall_curve(
     ...     y_true, y_scores)
     >>> precision
-    array([0.66666667, 0.5       , 1.        , 1.        ])
+    array([0.5       , 0.66666667, 0.5       , 1.        , 1.        ])
```

Comment:
We should update the docstring of `thresholds` to define properly `n_thresholds`.